.travis.yml

```diff
@@ -1,10 +1,14 @@
+dist: xenial
 language: python
 sudo: false
-python: 2.7
-env:
-    - TOX_ENV=py27
-    - TOX_ENV=py27-flake8
+matrix:
+  include:
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - env: TOXENV=flake8
```

Comment:
This isn't doing what you might think it's doing. This is running the tests, not checking syntax with flake8. We should change that test env in tox to use python 3, and then change this to match the `[py37-flake8]` env name. You can see in the Travis output for the `[flake8]` env that it's actually just running the python tests.

Reply:
I updated the changes I made. Let me know, if it addressed your suggestion.